_data/members.yml

```diff
@@ -197,20 +204,93 @@ medical_doctors:
     picture: 'lFanton.jpg'
     speciality: "Forensic scientist"
 
+  - firstname: 'Thimothée'
+    lastname: 'Jackesson'
+    institution: 'HCL - MCU-PH'
+    picture: 'tJackesson.jpg'
+    speciality: "Neurosurgeon"
+
   - firstname: 'Jean-Christophe'
     lastname: 'Richard'
     institution: 'HCL - PUPH'
     picture: 'jcRichard.jpg'
     speciality: "Reanimation"
     expertise: "Pulmonary imaging"
-    
-  - firstname: 'Thimothée'
-    lastname: 'Jackesson'
+        
+  - firstname: 'Salim'
+    lastname: 'Si-Mohamed'
     institution: 'HCL - MCU-PH'
-    picture: 'tJackesson.jpg'
-    speciality: "Neurosurgeon"
+    picture: 'sSiMohamed.jpg'
+    speciality: "Radiologist"
+    expertise: "Pulmonary imaging"        
+        
+phd_first:
+
+  - firstname: 'Théo'
+    lastname: 'Di Piazza'
+    institution: 'University Lyon 1'
+    picture: 'tDipiazza.jpg'
+    subject: 'Support for image-guided radiology report writing'
+    interests: "Large Language Model, Deep learning"
+    supervisors: L. Boussel   
+    
+  - firstname: 'Alae-eddine'
+    lastname: 'El Barkaoui'
+    institution: 'INSA, University of Lyon'
+    picture: 'icon_phd.jpg'
+    subject: 'Deep learning for automatic characterization of neurodegenerative diseases using multiparametric SPECT/CT brain images'
+    interests: "Neuro imaging, deep learning"
+    supervisors: T. Grenier, A. Flaus    
+
+  - firstname: 'Celia'
+    lastname: 'Goujat'
+    institution: 'INSA, University of Lyon'
+    picture: 'cGoujat.jpg'
+    subject: 'Multi-modal cardiac MRI analysis using diffusion models'
+    interests: "Cardiac imaging, deep learning, diffusion model, foundation model"
+    supervisors: L. Boussel, O. Bernard
+
+  - firstname: 'Agnieszka'
+    lastname: 'Gutwinska'
+    institution: 'University Lyon 1'
+    picture: 'aGutwinska.jpg'
+    subject: 'Development of pulmonary applications of K-edge colour perfusion imaging in combination with a Spectral Photon Counting Scanner and dedicated contrast agents'
+    interests: "Spectral scanner, quantitative imaging"
+    supervisors: S. Si-Mohamed    
+    
+  - firstname: 'Séréna'
+    lastname: 'Hariga'
+    institution: 'INSA, University of Lyon'
+    picture: 'sHariga.jpg'
+    subject: 'Hyperspectral Single-pixel Image reconstruction using nonnegative matrix factorization and deep learning'
+    interests: "Hyperspectral imaging, deep learning, Matrix factorization"
+    supervisors: J. Cohen, D. Friboulet, N. Ducros    
+    
+  - firstname: 'Thierry'
+    lastname: 'Judge'
+    institution: 'INSA, University of Lyon'
+    picture: 'tJudge.jpg'
+    subject: 'Analysis of cardiac tissue deformation using deep learning from a very large-scale database'
+    interests: "Cardiac imaging, deep learning, image simulation"
+    supervisors: O. Bernard, N. Duchateau, P.M. Jodoin    
+    
+  - firstname: 'Robin'
+    lastname: 'Trombetta'
+    institution: 'INSA, University of Lyon'
+    picture: 'rTrombetta.jpg'
```

Comment:
The picture you added for Robin has a JPEG extension (not JPG), so the link to the image is broken.

Either rename the file (which I would recommend for consistency's sake), or fix the reference here.